AElf.Kernel.SmartContractExecution/Application/IBlockAttachService.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Threading.Tasks;
+using AElf.Kernel.Blockchain.Application;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Volo.Abp.DependencyInjection;
+
+namespace AElf.Kernel.SmartContractExecution.Application
+{
+    public interface IBlockAttachService
+    {
+        void AttachBlock(Block block);
+    }
+
+    public class BlockAttachService : IBlockAttachService, ITransientDependency
+    {      
+        private readonly IBlockchainService _blockchainService;
+        private readonly IBlockchainExecutingService _blockchainExecutingService;
+        private readonly ITaskQueueManager _taskQueueManager;
```

Comment:
I have changed my mind, move queue mananger out of the service.

Reply:
What is your idea and how to use it now?